### PR TITLE
Migration for Startpage url updates for CB Assignments

### PR DESCRIPTION
**Pull request by h-kataria**
UI PR https://github.com/ManageIQ/manageiq-ui-classic/pull/7049

### CODE REVIEW

File: db/migrate/20200520182548_update_chargeback_assignments_startpage.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class UpdateChargebackAssignmentsStartpage < ActiveRecord::Migration[5.2]
+  class User < ActiveRecord::Base
+    serialize :settings, Hash
+  end
+
+  def up
+    say_with_time 'Updating starting page for users who had chargeback assignments set' do
+      User.select(:id, :settings).each do |user|
```
Comment (bdunne) [inline db/migrate/20200520182548_update_chargeback_assignments_startpage.rb:8]: This should probably be scoped with `in_my_region`

File: db/migrate/20200520182548_update_chargeback_assignments_startpage.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        if user.settings&.dig(:display, :startpage) == 'chargeback_assignments/explorer'
+          user.update!(:settings => user.settings.deep_merge(:display => {:startpage => 'chargeback_assignment'}))
+        end
+      end
+    end
+  end
+
+  def down
+    say_with_time 'Updating starting page for users who had non-explorer chargeback assignments pages set' do
+      User.select(:id, :settings).each do |user|
```
Comment (bdunne) [inline db/migrate/20200520182548_update_chargeback_assignments_startpage.rb:18]: same here

Comment (h-kataria) [reply db/migrate/20200520182548_update_chargeback_assignments_startpage.rb]: @bdunne made requested changes.

**Comment by simaishi** [comment]
Jansa backport details:

```
$ git log -1
commit 122df7ecb2bf94f57b0c4ad53e6469f9aafe434c
Author: Oleg Barenboim <chessbyte@gmail.com>
Date:   Fri May 22 15:24:52 2020 -0400

    Merge pull request #486 from h-kataria/cb_assignment_start_page_update

    Migration for Startpage url updates for CB Assignments

    (cherry picked from commit b699755b37cb29042820ccd2f79e93178310ca6f)
```

**Pull request merged**